newIDE/app/src/ProjectManager/index.js

```diff
@@ -789,472 +841,622 @@ export default class ProjectManager extends React.Component<Props, State> {
       onReloadEventsFunctionsExtensions,
       onInstallExtension,
     } = this.props;
-    const { renamedItemKind, renamedItemName, searchText } = this.state;
+    const {
+      renamedItemKind,
+      renamedItemName,
+      searchText,
+      openedExtensionShortHeader,
+      openedExtensionName,
```

Comment:
it's a bit confusing to me as to why we need both the `openedExtensionName` and the `openedExtensionName` in the state.
Aren't they redundant?

Reply:
`openedExtensionShortHeader` is the object returned by the API that can be considered as the truth whereas `openedExtensionName` is the name of the extension in the project, name that can be modified by the user.

Reply:
oh right, ok got it.